scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -171,6 +171,24 @@
                       (catch Exception e
                         (log/debug e "Error reading a string from mesos status data. Is it in the format we expect?")))))})
 
+(defn update-reason-histos!
+  "Updates histograms for run time, cpu time, and memory time,
+  where the histograms have the failure reason in the title"
+  [db mesos-reason instance-runtime {:keys [cpus mem]}]
```

Comment:
Should we check whether `reason` is `nil`?

Reply:
`mesos-reason->cook-reason-entity-id` returns the `:mesos-unknown` reason if the reason from mesos is `nil`